scan.sh

```diff
@@ -1,35 +1,62 @@
 #!/bin/bash
-set -eux
+set -euo pipefail
```

Comment:
this causes line 6 to fail, depending on whether or not the lowercase conversion matches certain chars. Hence turning it on the line after